perf/perf_genericevents.py

```diff
@@ -46,7 +48,34 @@ def generic_events(x):
         'branch-loads': "0x10068",
         'dTLB-load-misses': "0x300fc",
         'iTLB-load-misses': "0x400fc"
-    }.get(x, 9)
+    },
+        'p9': {
+        'cpu-cycles': "0x1e",
+        'stalled-cycles-frontend': "0x100f8",
+        'stalled-cycles-backend': "0x1e054",
+        'instructions': "0x02",
+        'branch-instructions': "0x4d05e",
+        'branch-misses': "0x400f6",
+        'cache-references': "0x100fc",
+        'cache-misses': "0x2c04e",
+        'L1-dcache-load-misses': "0x2c04e",
+        'L1-dcache-loads': "0x100fc",
+        'L1-dcache-prefetches': "0x20054",
+        'L1-dcache-store-misses': "0x300f0",
+        'L1-icache-load-misses': "0x200fd",
+        'L1-icache-loads': "0x4080",
+        'L1-icache-prefetches': "0x488c",
+        'LLC-load-misses': "0x300fe",
+        'LLC-loads': "0x4c042",
+        'LLC-prefetches': "0x4e052",
+        'LLC-store-misses': "0x26880",
+        'LLC-stores': "0x16880",
+        'branch-load-misses': "0x400f6",
+        'branch-loads': "0x4d05e",
+        'dTLB-load-misses': "0x300fc",
+        'iTLB-load-misses': "0x400fc"
+    }}
```

Comment:
can these dict generated  from a cfg file as i think editing test code in not good